code/game/mecha/equipment/weapons/ranged/laser_weapons.dm

```diff
@@ -50,34 +52,42 @@
 	name = "\improper Multi-laser Cannon"
 	desc = "Four solaris laser cannons welded together to achieve a higher firerate."
 	icon_state = "mecha_auto_laser"
-	energy_drain = 180
+	matter = list(MATERIAL_STEEL = 10, MATERIAL_PLASTEEL = 5, MATERIAL_SILVER  =1)
+	energy_drain = 500
 	projectile = /obj/item/projectile/beam/heavylaser
 	projectiles = 8
 	projectiles_per_shot = 2
-	fire_cooldown = 1
+	fire_cooldown = 5 //so we dont fire all our beams at the exact same instant. Remember! We track once a target is clicked!
 	fire_sound = 'sound/weapons/lasercannonfire.ogg'
 
 /obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/ion
-	equip_cooldown = 40
+	equip_cooldown = 40 //this is fine. This is a very dangerous tool.
 	name = "mkIV ion heavy cannon"
 	icon_state = "mecha_ion"
-	energy_drain = 120
-	projectile = /obj/item/projectile/ion
-	fire_sound = 'sound/weapons/Laser.ogg'
-	price_tag = 300
+	/obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/laser/heavy
+	equip_cooldown = 15
+	name = "\improper CH-LC \"Solaris\" laser cannon"
+	icon_state = "mecha_laser"
+	matter = list(MATERIAL_STEEL = 25, MATERIAL_PLASTEEL = 15, MATERIAL_SILVER  =10)
+	energy_drain = 500 //same as multi-laser, VERY dangerous and VERY powerful
```

Comment:
The Multi-laser cost more power to operate because it is literally 4 of these stuck together to compensate for their individual low firerate.

Reply:
Ah...this was also a mistake, that comment was meant to be over the ion cannons energy change. Will need to look over this in the morning and see what else i goofed up on